cloudferrylib/os/image/glance_image.py

```diff
@@ -190,10 +190,10 @@ def image_exists(self, image_id):
             except glance_exceptions.HTTPNotFound:
                 return False
 
-    def get_image_list(self):
+    def get_image_list(self, is_filtering=True):
         images = self.glance_client.images.list(filters={"is_public": None})
```

Comment:
the design, you added `is_filtering` parameter to lots of functions, but is's using only in one place. also Anton wrote below, what is wrong